docs/reference/NuGet-Client-SDK.md

```diff
@@ -111,14 +125,14 @@ Read a package from a file stream using [`NuGet.Packaging`](https://www.nuget.or
 
 You can find examples and documentation for some of the API in the following blog series by Dave Glick, published 2016:
 
-- [Exploring the NuGet v3 Libraries, Part 1: Introduction and concepts](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-1)
-- [Exploring the NuGet v3 Libraries, Part 2: Searching for packages](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-2)
-- [Exploring the NuGet v3 Libraries, Part 3: Installing packages](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-3)
+* [Exploring the NuGet v3 Libraries, Part 1: Introduction and concepts](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-1)
+* [Exploring the NuGet v3 Libraries, Part 2: Searching for packages](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-2)
+* [Exploring the NuGet v3 Libraries, Part 3: Installing packages](http://daveaglick.com/posts/exploring-the-nuget-v3-libraries-part-3)
 
 > [!Note]
 > These blog posts were written shortly after the **3.4.3** version of the NuGet client SDK packages were released.
 > Newer versions of the packages may be incompatible with the information in the blog posts.
 
 Martin Björkström did a follow-up blog post to Dave Glick's blog series where he introduces a different approach on using the NuGet Client SDK to install NuGet packages:
 
-- [Revisiting the NuGet v3 Libraries](https://martinbjorkstrom.com/posts/2018-09-19-revisiting-nuget-client-libraries)
+* [Revisiting the NuGet v3 Libraries](https://martinbjorkstrom.com/posts/2018-09-19-revisiting-nuget-client-libraries)
```

Comment:
I clicked on "Fix All Known Formatting Problems" in VS Code, which resulted in this change. I am happy to revert it if the team thinks this change should not be part of this PR.

I didn't see any difference in the rendered content between this change in the preview URL and the production URL.

[Preview URL for this section](https://review.learn.microsoft.com/en-us/nuget/reference/nuget-client-sdk?branch=pr-en-us-3176#third-party-documentation)

[Production URL for this section](https://learn.microsoft.com/en-us/nuget/reference/nuget-client-sdk#third-party-documentation)